Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,18 @@
 import React from 'react';
 import './Card.css';
+
+const CardContent = () => {
+	return (
+		<div className="content">
+			<CardTitle />
+			<Copy />
+			<Link />
+		</div>
+	)
+}
+
+const CardTitle = () => <h4>Get started with React</h4>
+const Copy = () => <p>React makes it painless to create interactive UIs. Design simple views for each state in your application.</p>
+const Link = () => <p className="link">reactjs.org</p>
```

Comment:
i'm guessing this is an experiment, but since each of these components only contain a single html element there's no reason to declare them as components, they could just be written directly into the JSX for CardContent.

```
const CardContent = () => {
	return (
		<div className="content">
			<h4>Get started with React</h4>
			<p>React makes it painless to create interactive UIs. Design simple views for each state in your application.</p>
			<p className="link">reactjs.org</p>
		</div>
	)
}
```